src/stores/chatExtras.js

```diff
@@ -130,6 +140,16 @@ export const useChatExtrasStore = defineStore('chatExtras', {
 			Vue.delete(this.parentToReply, token)
 		},
 
+		/**
+		 * Restore chat input from the browser storage and save to store
+		 *
+		 * @param {string} token The conversation token
+		 */
+		restoreChatInput(token) {
+			const chatInput = BrowserStorage.getItem('chatInput_' + token)
+			Vue.set(this.chatInput, token, chatInput ?? '')
```

Comment:
Although you has empty strings only from here. Don't produce it here as well?

Reply:
Empty string is needed in the component so we can't render null or undefined, but it will be better if we render it for getChatInput and not store it.